frontend/src/cards/AgeAdjustedTableCard.tsx

```diff
@@ -92,7 +92,7 @@ export function AgeAdjustedTableCard(props: AgeAdjustedTableCardProps) {
     config.metricId.includes("ratio")
   );
 
-  const locationName = props.fips.getSentenceDisplayName();
+  const locationName = `in ${props.fips.getSentenceDisplayName()}`;
```

Comment:
very minor thing, but for naming a variable like this, I like to use something that indicates the preposition "in" is coming along... using `locationName` implies to me it's only the name. I might use something like `locationPhrase` or something like that so it's clear there is more than just the name itself included in the variable string